backend/app.js

```diff
@@ -49,10 +49,10 @@ app.listen(app.get("port"), function() {
   console.log("Listening on port", app.get("port"));
 });
 
-export function updateLumensCache() {
-  lumens.updateApiLumens();
-  lumensV2.updateApiLumens();
-  lumensV3.updateApiLumens();
+export async function updateLumensCache() {
+  await lumens.updateApiLumens();
+  await lumensV2.updateApiLumens();
+  await lumensV3.updateApiLumens();
```

Comment:
💡 You could wait for them in parallel using `await Promise.all(...)`.

Reply:
I actually need it to be not in parallel, so that all the calls don't hit horizon at the same time :/ but agreed otherwise that would be cleaner